bin/list.json

```
@@ -5,19 +5,25 @@
       "version": "0.1.1",
       "build": "commit.6ff4cd6",
       "longVersion": "0.1.1+commit.6ff4cd6",
+      "sha256": "0xfc54135edfe07d835216f87fa6c3b9049e1415c2825027b926daad018e09269f",
       "keccak256": "0xd8b8c64f4e9de41e6604e6ac30274eff5b80f831f8534f0ad85ec0aff466bb25",
       "urls": [
-        "bzzr://8f3c028825a1b72645f46920b67dca9432a87fc37a8940a2b2ce1dd6ddc2e29b"
+        "bzzr://8f3c028825a1b72645f46920b67dca9432a87fc37a8940a2b2ce1dd6ddc2e29b",
+        "https://ethereum.github.io/solc-bin/bin/soljson-v0.1.1+commit.6ff4cd6.js",
+        "http://ethereum.github.io/solc-bin/binsoljson-v0.1.1+commit.6ff4cd6.js"
```

Shouldn't this be
```suggestion
        "http://ethereum.github.io/solc-bin/bin/soljson-v0.1.1+commit.6ff4cd6.js"
```
?

---

0xa1291d7e34614eb11AD305f299f062cAB485e0E5